OG/OG/AST/Terminals/NumberNode.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OG.AST;
+
+namespace OG.AST.Terminals
+{
+    public class NumberNode<T> : ASTNode
```

Comment:
kan overvejes, at der sættes en restriktion med "where T is int, float, long, decimal" eller lign. konstraint (selvom det ikke er vanvittigt vigtigt)